assets/js/script.js

```diff
@@ -30,12 +30,22 @@ function handleFormSubmit(event) {
      //put in tr
      tableRow.append(tableData);
      };
-  
+     var buttonTD = $('<td>')
+     var deleteButton = $('<button>');
+     deleteButton.attr('type', 'button') 
+     deleteButton.attr('class', 'btn btn-danger delete fa fa-trash') 
```

Comment:
Love the font awesome here

Reply:
Yeah, thanks google! I'd used them in the past